src/test/resources/assets/modid/bc/MMPL-1.0.txt.txt

```diff
@@ -0,0 +1,90 @@
+As per: https://www.mod-buildcraft.com/MMPL-1.0.txt
+
```

Comment:
This file has a duplicate ending (`.txt.txt`)

Reply:
Oops